scripts/localization_extract

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
```

Comment:
Runs the built in localization extract tool on all directories with source code. -s instructs it to look for `WMFLocalizedString` in addition to `NSLocalizedString`